AndersenHWs/src/main/java/org/example/ConcertTicket.java

```diff
@@ -0,0 +1,136 @@
+package org.example;
+
+import java.util.Objects;
+
+public class ConcertTicket extends Ticket{
+    public static Double price;
```

Comment:
hy its pubic static, not just private as other ones?

Reply:
*** Add the ability to save ticket price. This should be a new variable, think carefully about the type.

I thought that public static would do the job. In the Ticket Service class, I wrote "ConcertTicket.price = 3000.0;", and thought that making price static -  the ability to save ticket price.